dockerfiles/zookeeper/Dockerfile

```diff
@@ -1,22 +0,0 @@
-ARG IMAGE_VER=3.4.11
```

Comment:
Note: I'd love to use this in the reworked Dockerfile, but it looks like the Docker version on my system is too old to support this 👎